docs/authentication/config.mdx

```diff
@@ -59,6 +61,8 @@ const response = await fetch("http://localhost:3000/api/pages", {
 });
 ```
 
+Our authentication strategies ensure uniform access control across all strategies. This enables you to utilize your existing access control configurations with both API keys and the standard email/password authentication. This consistency can aid in maintaining granular control over your API keys.
```

Comment:
"Our authentication strategies ensure uniform access control across all strategies" sounds a bit weird, with 2x "strategies". Maybe this is better:

"You will have the same, uniform access control across all our authentication strategies."

or

"Payload ensures that the same, uniform access control is used across all authentication strategies"

I like everything else!

Reply:
@AlessioGr Good catch, thanks for the suggestion! I updated.